docs/getting-started.md

```diff
@@ -115,9 +115,34 @@ The above code will invoke the `lookupTerm` function that we wish to test with a
 application's thesaurus. Each `assertEqual` function in the test library - along with every other assertion function - 
 will return a success or failure. The test then returns an array of these successes and failures.
 
+## Configuring a connection to MarkLogic
```

Comment:
Realized that this section is needed so that the user knows what to configure in ml-gradle for connecting. This is where things get a little wonky in terms of what's specific to marklogic-unit-test and what's specific to ml-gradle.